### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by pberdyuhin**
Switch base docker image to rhel-ubi8

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,20 @@
-FROM alpine:3.7
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```
Comment (kiamran) [inline Dockerfile:1]: set tag excplicitly

File: Makefile [hunk 2]
```diff
@@ -69,8 +69,9 @@ dockerhub-login:
 	docker login -u $(DOCKER_USERNAME) -p $(DOCKER_PASSWORD)
 
 version-set:
-	@sed -i '' 's#$(DOCKER_IMAGE_NAME):$(PREVIOUS_VERSION)#$(DOCKER_IMAGE_NAME):$(VERSION)#g' deployment/docker-compose/docker-compose.yaml && \
-	sed -i '' 's#$(DOCKER_IMAGE_NAME):$(PREVIOUS_VERSION)#$(DOCKER_IMAGE_NAME):$(VERSION)#g' e2e/docker-compose.yaml && \
+	@sed -i '' 's#$(DOCKER_IMAGE_NAME):.*#$(DOCKER_IMAGE_NAME):$(VERSION)#g' deployment/docker-compose/docker-compose.yaml && \
+	sed -i '' 's#$(DOCKER_IMAGE_NAME):.*#$(DOCKER_IMAGE_NAME):$(VERSION)#g' e2e/docker-compose.yaml && \
+	sed -i '' 's#version=.*#version="$(VERSION)" \\#g' Dockerfile && \
```
Comment (kiamran) [inline Makefile:74]: may be better to set version using command line args when building docker image ?

File: Makefile [hunk 3]
```diff
@@ -69,8 +69,9 @@ dockerhub-login:
 	docker login -u $(DOCKER_USERNAME) -p $(DOCKER_PASSWORD)
 
 version-set:
```
Comment (kiamran) [inline Makefile:71]: i am thinking if we can remove this `version-set` command.. can we adjust version on the flight ? without making this command required to be executed

File: Dockerfile (hunk 1 above)
Comment (pberdyuhin) [reply Dockerfile]: set to ubi-minimal:8.3

File: Makefile (hunk 3 above)
Comment (pberdyuhin) [reply Makefile:71]: I think we should save this option. It's convenient for changing tags in docker-compose files for testing environments. It isn't  necessary step during deploing

File: Makefile (hunk 2 above)
Comment (pberdyuhin) [reply Makefile]: Good catch! Changed it as you mentioned.

**Comment by kiamran** [comment]
please, fix conflicts and we are good to go

**Pull request merged**